hieradata/site/ls/role/foreman.yaml

```diff
@@ -40,6 +40,12 @@ dhcp::pools:
     gateway: "139.229.147.254"
     range: "139.229.147.225 139.229.147.253"  # ~ /27
     search_domains: "%{alias('dhcp::dnsdomain')}"
+  ComCam:
+    network: "139.229.150.0"
+    mask: "255.255.255.128"
+    gateway: "139.229.150.127"
+    range: "139.229.150.1 139.229.150.100"
```

Comment:
This range is probably much to large.  I would suggest cutting it down to the upper /27 or /28.